libs/core/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.0.61",
   "description": "Core styles and helpers for the Kirby Design System.",
   "type": "module",
-  "main": "dist/index.js",
-  "module": "dist/index.js",
-  "types": "dist/types/index.d.ts",
```

Comment:
We can simplify this, as there is no reason to have a dist folder inside our published package. In our publish script we copy the package.json to the same level as our index.js files already, so there is no need to nest things inside a dist folder. Now it just looks like this when installed:
![image](https://github.com/kirbydesign/designsystem/assets/42470636/ed280bdd-4a50-4b0d-9801-01c97f1d9b41)